ledger/metrics/src/main/scala/com/daml/metrics/CommandMetrics.scala

```diff
@@ -3,23 +3,100 @@
 
 package com.daml.metrics
 
+import com.daml.metrics.MetricDoc.MetricQualification.Debug
 import com.daml.metrics.MetricHandle.{Counter, Meter, Timer}
 
 import com.codahale.metrics.{MetricRegistry}
 
 class CommandMetrics(override val prefix: MetricName, override val registry: MetricRegistry)
     extends MetricHandle.Factory {
+
+  @MetricDoc.Tag(
+    summary = "The time to validate a Daml command.",
+    description = """The time to validate a submitted Daml command before is fed to the
+                    |interpreter.""",
+    qualification = Debug,
+  )
   val validation: Timer = timer(prefix :+ "validation")
+
+  @MetricDoc.Tag(
+    summary = "The time to fully process a Daml command.",
+    description = """The time to validate and interpret a command before it is handed over to the
+                    |synchronization services to be finalized (either committed or rejected).""",
+    qualification = Debug,
+  )
   val submissions: Timer = timer(prefix :+ "submissions")
+
+  @MetricDoc.Tag(
+    summary =
+      "The number of the Daml commands that are currently being handled by the ledger api server.",
+    description = """The number of the Daml commands that are currently being handled by the ledger
+                    |api server (including validation, interpretation, and handing the transaction
+                    |over to the ledger).""",
+    qualification = Debug,
+  )
   val submissionsRunning: Meter = meter(prefix :+ "submissions_running")
 
+  @MetricDoc.Tag(
+    summary = "The number of Daml commands that failed in interpretation.",
+    description = """The number of Daml commands that have been rejected by the interpreter
+                    |(e.g. non badly authorized action).""",
```

Comment:
```suggestion
                    |(e.g. badly authorized action).""",
```
I must have overlooked it on the sheet